widget/src/App.svelte

```diff
@@ -0,0 +1,6 @@
+<script lang="ts">
+  export let name: string = "friend";
+  console.log(arguments);
+</script>
+
+<h1>Hello {name}!</h1>
```

Comment:
`parcel-transformer-svelte3-plus` is going to automatically enable `svelte-preprocess` for us (if it's installed), so we can use the template tag feature: https://github.com/sveltejs/svelte-preprocess#template-tag

So, I'd suggest wrapping the HTML in `.svelte` in `<template>` tags.